Develappers.BillomatNet/Helpers/MappingHelpers.cs

```diff
@@ -86,5 +86,19 @@ public static object ParsePropertyValue(PropertyType type, string value)
 
             return value;
         }
+
+        public static string ParsePropertyValue(PropertyType type, object value)
```

Comment:
hier bitte nicht einfach einen cast auf string machen, sondern vorher value auf "is bool" prüfen